docs/developers/guides/gas/gas-fees.mdx

```diff
@@ -36,12 +36,67 @@ and gas used.
 `linea_estimateGas` is the recommended method for estimating gas on Linea. See our 
 [reference page](../../reference/api/linea-estimategas.mdx) for more information. 
 
-Linea also supports [`eth_estimateGas`](https://docs.infura.io/api/networks/linea/json-rpc-methods/eth_estimategas),
-[`eth_gasPrice`](https://docs.infura.io/api/networks/linea/json-rpc-methods/eth_gasprice), and
-[`eth_feeHistory`](https://docs.infura.io/api/networks/linea/json-rpc-methods/eth_feehistory).
+Linea also supports:
+- [`eth_estimateGas`](https://docs.infura.io/api/networks/linea/json-rpc-methods/eth_estimategas)
+- [`eth_gasPrice`](https://docs.infura.io/api/networks/linea/json-rpc-methods/eth_gasprice)
+- [`eth_feeHistory`](https://docs.infura.io/api/networks/linea/json-rpc-methods/eth_feehistory).
+
+## Gas pricing
+
+The gas price you retrieve using methods such as `linea_estimateGas` is based on values calculated
+by the [coordinator](../../../architecture/stack/coordinator/index.mdx), and uses the gas price of 
+the previous block with a profitability multiplier applied.
+
+Firstly, the coordinator records:
```

Comment:
I think notion of Coordinator doesn't help clarity. I think in this explanation it's better omit such details and focus on what's visible from user's perspective. What and user can see is extraData, e.g.
```
curl -sH "Content-Type: application/json" --data "{\"jsonrpc\":\"2.0\",\"method\":\"eth_getBlockByNumber\",\"params\":[\"latest\", false],\"id\":42}" https://rpc.sepolia.linea.build | jq .result.extraData
```

Reply:
thank you — reframed to simplify